libsolidity/ast/Types.h

```diff
@@ -599,7 +597,7 @@ class FixedBytesType: public Type
 	bool leftAligned() const override { return true; }
 	bool isValueType() const override { return true; }
 
-	std::string toString(bool) const override { return "bytes" + dev::toString(m_bytes); }
+	std::string toString(bool) const override { return "bytes" + ::solidity::util::toString(m_bytes); }
```

Comment:
```suggestion
	std::string toString(bool) const override { return "bytes" + solidity::util::toString(m_bytes); }
```

Any reason for the ``::``? Or for also ``solidity`` for that matter?

Reply:
Not that a single one makes any difference, but the idea is to start the name resolving at the root namespace, hence, it *should* be a few ms quicker (because it doesn't need to recursively ascend upwards), but again, not that it should be anything like noticeable when building (I didn't measure) :D